sw/device/lib/testing/test_rom_ext/BUILD

```diff
@@ -7,21 +7,21 @@ package(default_visibility = ["//visibility:public"])
 load("//rules:opentitan.bzl", "OPENTITAN_CPU")
 
 cc_library(
-    name = "common",
+    name = "test_rom_ext",
     srcs = [
-        "flash_crt.S",
-        "ibex_interrupt_vectors.S",
+        "test_rom_ext_start.S",
+        "test_rom_ext_irq_vector.S",
     ],
     copts = [
         "-nostdlib",
         "-ffreestanding",
     ],
     linkopts = [
-        "-T $(location flash_link.ld)",
+        "-T $(location test_rom_ext.ld)",
     ],
     target_compatible_with = [OPENTITAN_CPU],
     deps = [
-        "flash_link.ld",
+        "test_rom_ext.ld",
         "//hw/top_earlgrey/sw/autogen:linker_script",
         "//sw/device:info_sections",
```

Comment:
This target also appears to depend on
`
        "//sw/device/lib/crt",
        "//sw/device/lib/testing/test_framework:ottf_isrs",
`
to link properly.

Reply:
Thanks @drewmacrae ! Fixed.